docs/developer/template_switches.asciidoc

```diff
@@ -0,0 +1,118 @@
+// to display images directly on GitHub
+ifdef::env-github[]
+:imagesdir: ../images
+endif::[]
+
+Creating a new Switch via a Template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Please use "=" syntax for section titles, see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles

Reply:
done